google-cloud-service_directory/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Release History
```

Comment:
Do we want to write something like 'initial release of version v1beta1" here?

Reply:
The changelog will get updated by releasetool when we do the actual release.